docs/bouwblokken/fundamentele rechten/non-discriminatie.md

```diff
@@ -1,44 +1,201 @@
+In de Nederlandse taal spreken we dan van oneerlijkheid of onrechtvaardigheid (of in positieve zin van respectievelijk fairness, eerlijkheid en rechtvaardigheid). 
+Oneerlijk gedrag van algoritmes en AI-systemen kunnen leiden tot het niet naleven van gevestigde overtuigingen en normen, wat voorkeursbehandeling en discriminatie in de hand kan werken. 
+
+[^5]: Zie [NEN-EN-ISO/IEC 22989:2023 en](https://www.nen.nl/nen-en-iso-iec-22989-2023-en-312642) [^3] 
+
+Ongerechtvaardigde algoritmische bias is vaak een gevolg van bepaalde processen of systemen die gebruikt worden op zo'n wijze dat bepaalde groepen bevoordeeld worden en andere groepen benadeeld worden.
+Dit is vaak geen bewuste vorm van vooringenomenheid, maar kan bijvoorbeeld ontstaan doordat de meerderheid bestaande regels of normen volgt.
+
+!!! note "Opmerking"
+    Wanneer we in de context van algoritmes spreken over bias, wordt vaak deze betekenis van bias bedoeld. 
+    Dit heeft als gevolg dat wanneer de term *bias* in het algoritmekader afzonderlijk wordt gebruikt, deze betekenis wordt bedoeld. Alle andere betekenissen van bias zullen specifiek benoemd worden.
+
+#### Statistische bias
+Statistische bias wordt gedefinieerd als *een consistente numerieke afwijking van een schatting ten opzichte van de werkelijke onderliggende waarde*. 
+Deze vorm van bias komt bijvoorbeeld voor wanneer er een steekproef wordt gebruikt die niet representatief is voor de populatie, en de schattingen op basis van de steekproef vervolgens systematisch afwijken van de werkelijke waarde.
+Statistische bias duidt op een systematische fout die gemaakt wordt door het algoritme. Deze fouten kunnen vrij zijn van vooroordelen of discriminerende bedoelingen. 
+Voorbeelden van statistische bias zijn meetfouten (measurement bias), foute data of data op een te simpele manier representeren (representatie bias).
+
+#### Menselijke bias 
+Menselijke bias omvat de systematische fouten in het menselijk denken. 
+Deze menselijke vooroordelen zijn vaak impliciet van aard en hebben betrekking op de manier waarop een individu bepaalde informatie waarneemt en verwerkt om bijvoorbeeld een beslissing te nemen. 
+In de context van algoritmes kan deze vorm van bias voorkomen tijdens de ontwikkeling van een algoritme, waarbij menselijke vooroordelen verwerkt kunnen worden in het algoritme (bijvoorbeeld wanneer er gebruik wordt gemaakt van rekenregels).
+Daarnaast kan menselijke bias voorkomen tijdens de afhandeling, wanneer een door een algoritme geselecteerde groep handmatig wordt afgehandeld. 
+Voorbeelden van vormen menselijke bias is wanneer er voorkeur wordt geven aan de voorspellingen van een algoritme die reeds bestaande overtuigigngen bevestigen (confirmation bias/bevestigingsbias), of wanneer mensen de neiging hebben om voorkeur te geven aan suggesties die door het algoritme worden gedaan (automatiseringsbias).
```

Comment:
Een ander voorbeeld is dat mensen personen soms ongelijk behandeling op basis van gender of afkomst.